cristin-import/src/main/java/no/unit/nva/cristin/lambda/EventEmitter.java

```diff
@@ -116,11 +118,15 @@ private PutEventsRequestEntry createPutEventRequestEntry(T eventDetail) {
                    .resources(invokingFunctionArn)
                    .detailType(detailType)
                    .time(Instant.now())
-                   .detail(eventDetail.toJsonString())
+                   .detail(toJson(eventDetail))
```

Comment:
We do not longer expect the generic class to extend JsonSerializable (own interface) because we want to serialize JsonNodes